src/component/style.js

```diff
@@ -11,6 +11,7 @@ export const colors = {
   blueSig: '#002FFF',
   glas: 'rgba(255,255,255,0.13)',
   glasDark: 'rgba(255,255,255,0.08)',
+  darkText: '#4A4A4A',
```

Comment:
I've added `blackText` in https://github.com/lightninglabs/lightning-app/commit/4b33b5175eb9dd49ab2d765349be1167566dc9f1 to stay consistent with our naming.